test/unit/style.js

```diff
@@ -0,0 +1,36 @@
+import Style from 'Core/Style';
+import assert from 'assert';
+
+describe('Style', function () {
+    const style1 = new Style();
+    style1.point.color = 'red';
+    style1.fill.color = 'blue';
+    style1.stroke.color = 'black';
+
+    it('Copy style', () => {
+        const style2 = new Style().copy(style1);
+        assert.equal(style1.point.color, style2.point.color);
+        assert.equal(style1.fill.color, style2.fill.color);
+        assert.equal(style1.stroke.color, style2.stroke.color);
+    });
+    it('Clone style', () => {
+        const style2 = style1.clone(style1);
+        assert.equal(style1.point.color, style2.point.color);
+        assert.equal(style1.fill.color, style2.fill.color);
+        assert.equal(style1.stroke.color, style2.stroke.color);
+    });
+    it('applyToHTML', () => {
+        const dom = { style: {} };
```

Comment:
You can use `document.createElement` which is present in `mock`.

Reply:
yes, thanks!